components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/DataFlowProcessingEngine.cs

```diff
@@ -126,18 +119,37 @@ public class DataFlowProcessingEngine : IProcessingEngine, IDisposable {
                     });
 
                 _sinkBlock = new ActionBlock<NetworkMessageModel[]>(
-                    async input => await _messageSink.SendAsync(input).ConfigureAwait(false),
+                    async input => {
+                        await _messageSink.SendAsync(input).ConfigureAwait(false);
+                        if (_batchTriggerInterval > TimeSpan.Zero) {
+                            _batchTriggerIntervalTimer.Change(_batchTriggerInterval, _batchTriggerInterval);
+                        }
+                    },
                     new ExecutionDataflowBlockOptions {
-                        CancellationToken = cancellationToken
+                        CancellationToken = cancellationToken,
```

Comment:
Remove comma

Reply:
thanks!

Reply:
I would argue that using a trailing comma is considered best practice. If we decide to add a new item, adding a comma and the new item in another line will result in two lines having changed.